tooling/build_changelog

```diff
@@ -0,0 +1,8 @@
+the_dir=$1
+: ${the_dir:=.}
+cd $the_dir
+pwd
+curl -s "https://raw.githubusercontent.com/release-depot/change/0.14.3/change" |
+sh -s -- init
+curl -s "https://raw.githubusercontent.com/release-depot/change/0.14.3/change" |
+sh -s --
```

Comment:
This looks like a repeat, am sure there is a reason but a comment or something explaining it would be helpful.

Reply:
Ok, I can add a comment on why this has to run twice, no problem.